conda.recipe\run_test fails when running the standard test suite

Following the [getting started guide](http://taxcalc.readthedocs.org/en/latest/contributor_guide.html), upon running `py.test -m "not requires_pufcsv"` a user is presented with 111 successes and 1 error:

```
conda.recipe\run_test.py:1: in (module)
    import taxcalc
E   ImportError: No module named `taxcalc`
```

I'm not super familiar with pytest and standard setups, but I have to imagine that this test is only meant to be run after Conda follows the recipe to build the package (especially since the above line is the entire test). 

When run from the taxcalc repo itself and not building, it makes sense that this would fail, correct? 

If so, should a contributor use one of pytest's [skipping methods](https://pytest.org/latest/skipping.html) to avoid this test?

--- a/.travis.yml
+++ b/.travis.yml
@@ -23,7 +23,7 @@ install:
 
 # command to run tests, e.g. python setup.py test
 script:
-  - python -c "import taxcalc"; coverage run -m py.test -v -m "not requires_pufcsv" --pep8 taxcalc/tests/
+  - python -c "import taxcalc"; coverage run -m py.test -v -m "not requires_pufcsv" --pep8
 
 after_success:
   - codecov

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,2 +1,3 @@
 [pytest]
 pep8ignore = E402
+testpaths = taxcalc

